### PR TITLE
Add ObjCons

### LINKED ISSUE #13: Add object construction expression
Currently, there is  no way to create object types directly.
So, we need to add the following syntax to the object construction expression
```
T(field1: value1, field2: value2, ..., fieldN: valueN)
```
where `T` is object type name.

An example is bellow:
```
type
    Vector2 = object
        x: float
        y: float
let a = Vector2(x: 3, y: 4.5)
```

### PR DESCRIPTION
**Pull request by Unit112358**

### CODE REVIEW

**Comment by chocobo333** [comment]
Can I ask you to add a unit test?

**Comment by Unit112358** [comment]
unit test for ObjCons?

**Comment by Unit112358** [comment]
I added unit test in src/parser.nim.
If something is lack or you have something that shound be added to unit test, please mention it.

File: src/parsers.nim [hunk 1]
```diff
@@ -751,3 +765,34 @@ func sum(l: List[int32]) -> int32:
...
+    )
+    let testProgram = Program(testSrc).get
+    import utils
+    debug testProgram
+    let 
+        s1 = testProgram.stmts[0]
+        s2 = testProgram.stmts[1]
+        s3 = testProgram.stmts[2]
+
+    debug s1.kind                # Expression
```
Comment (chocobo333) [inline src/parsers.nim:784]: You can use
```nim
assert s1.kind == Expression
```
instead

**Pull request merged**